### DISCUSSION TITLE
Add m_combinedFriction2 to btManifoldPoint.

### DISCUSSION THREAD

**Discussion by AccessViolationEnjoyer** [bulletphysics/bullet3#3509] [General]
You have `m_lateralFrictionDir2` and `m_contactMotion2`, but not `m_combinedFriction2`. ODE supports this through the `dContactMu2` mode flag and `dSurfaceParameters::mu2`.

**Comment by erwincoumans** [comment]
Thanks for the feedback. Yes, I'll look into adding this.
